### PR TITLE
Sync serverless ci

### PR DESCRIPTION
**Pull request by aliok**
/hold

Hold until https://github.com/openshift-knative/hack/pull/18 is merged and also until https://docs.ci.openshift.org/docs/how-tos/cluster-claim/#existing-cluster-pools has 4.12 pool

### CODE REVIEW

**Comment by aliok** [comment]
/unhold

I see 4.12 available in that document.

Gonna re-generate these though

**Comment by aliok** [comment]
Rebased

**Comment by aliok** [comment]
/hold

I don't know why those mirroring files are deleted. @pierDipi @matzew do you know?

The only thing I did is to re-run the generation after https://github.com/openshift-knative/hack/pull/18 is merged

File: core-services/image-mirroring/knative/mapping_knative_knative-nightly_eventing-kafka-broker_quay [hunk 1]
```diff
@@ -1,22 +0,0 @@
-registry.ci.openshift.org/openshift/knative-eventing-kafka-broker-dispatcher quay.io/openshift-knative/knative-eventing-kafka-broker-dispatcher
```
Comment (pierDipi) [inline core-services/image-mirroring/knative/mapping_knative_knative-nightly_eventing-kafka-broker_quay:1]: Why are these image mirroring files dropped?

**Review by pierDipi — COMMENTED** [review]
It is only changing eventing core files

File: core-services/image-mirroring/knative/mapping_knative_knative-nightly_eventing-kafka-broker_quay (hunk 1 above)
Comment (aliok) [reply core-services/image-mirroring/knative/mapping_knative_knative-nightly_eventing-kafka-broker_quay:1]: I don't know. I just re-ran the script and they're being dropped.

Gonna check again.

**Pull request closed without merging**